tests/conftest.py

```
@@ -134,18 +135,18 @@
+
     # Base port range: 8090-8199 (110 ports available)
     base_port = 8090
-    
+
     if worker_id == 'master':
         # Single worker/sequential execution
         port_offset = 0
     else:
         # Extract worker number from workerid (e.g., 'gw0', 'gw1', etc.)
         worker_num = int(worker_id.replace('gw', '')) if worker_id.startswith('gw') else 0
         port_offset = (worker_num * 10) + random.randint(0, 9)
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)